projectName-selenium/pom.xml

```diff
@@ -63,6 +63,49 @@
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-java</artifactId>
+            <version>${selenium.version}</version>
```

Comment:
- [ ] Version is unnecessary because version is defined in root pom `dependencyManagement`.